kernel: bump 6.6 to 6.6.82

graysky2 opened the pull request:
Changelog: https://cdn.kernel.org/pub/linux/kernel/v6.x/ChangeLog-6.6.82

All patches automatically rebased.

Added `CONFIG_MICROCODE_INITRD32=y` to x86: config-6.6[1]
    
1. https://git.kernel.org/pub/scm/linux/kernel/git/stable/linux.git/diff/arch/x86/Kconfig?id=v6.6.82&id2=v6.6.81

Build system: x86/64
Build-tested: bcm27xx/bcm2712
Run-tested: bcm27xx/bcm2712

graysky2 committed:
kernel: bump 6.6 to 6.6.81

Changelog: https://cdn.kernel.org/pub/linux/kernel/v6.x/ChangeLog-6.6.81

All patches automatically rebased.

For x86, CONFIG_MICROCODE_LATE_FORCE_MINREV was introduced in this bump
with hrecommendation to enable it[1].

1. https://web.git.kernel.org/pub/scm/linux/kernel/git/stable/linux.git/diff/arch/x86/Kconfig?id=v6.6.81&id2=v6.6.80

Build system: x86/64
Build-tested: bcm27xx/bcm2712, flogic/xiaomi_redmi-router-ax6000-ubootmod, ramips/tplink_archer-a6-v3
Run-tested: bcm27xx/bcm2712, flogic/xiaomi_redmi-router-ax6000-ubootmod, ramips/tplink_archer-a6-v3

Signed-off-by: John Audia <therealgraysky@proton.me>
Link: https://github.com/openwrt/openwrt/pull/18188
Signed-off-by: Hauke Mehrtens <hauke@hauke-m.de>

hauke commented:
i386 should have compile problems lets wait for 6.6.82, see https://lwn.net/Articles/1013395/

rockdrilla commented:
@graysky2 @hauke 6.6.82 is just released.

namiltd commented:
> @graysky2 @hauke 6.6.82 is just released.

Added new setting CONFIG_MICROCODE_INITRD32 to be added in target/linux/x86/config-6.6 (https://web.git.kernel.org/pub/scm/linux/kernel/git/stable/linux.git/commit/?h=linux-6.6.y&id=d4c860bb425d665cb2aa9c9f3e1c859aebf20f29)

graysky2 committed:
kernel: bump 6.6 to 6.6.82

Changelog: https://cdn.kernel.org/pub/linux/kernel/v6.x/ChangeLog-6.6.82

All patches automatically rebased.

Added CONFIG_MICROCODE_INITRD32=y to x86: config-6.6[1]

1. https://git.kernel.org/pub/scm/linux/kernel/git/stable/linux.git/diff/arch/x86/Kconfig?id=v6.6.82&id2=v6.6.81

Build system: x86/64
Build-tested: bcm27xx/bcm2712
Run-tested: bcm27xx/bcm2712

Signed-off-by: John Audia <therealgraysky@proton.me>
Link: https://github.com/openwrt/openwrt/pull/18188
Signed-off-by: Hauke Mehrtens <hauke@hauke-m.de>

graysky2 commented:
I have not done any testing with 6.6.82

namiltd commented:
`make kernel_oldconfig` adds `CONFIG_MICROCODE_INITRD32=y` to target/linux/x86/config-6.6

drujd commented:
6.6.82 working well on cherrypicked to master with minor changes and Turris Omnia.

namiltd commented:
Just add information in the description about CONFIG_MICROCODE_INITRD32 and I have no remarks.
